ee/tabby-webserver/src/handler.rs

```diff
@@ -84,6 +84,10 @@ impl WebserverHandle {
                 "/repositories",
                 repositories::routes(rs.clone(), ctx.auth()),
             )
+            .nest(
+                "/integrations",
+                integrations::routes(ctx.setting(), ctx.github_repository_provider()),
```

Comment:
```suggestion
                "/integrations/github",
                integrations::github::routes(ctx.setting(), ctx.github_repository_provider()),
```

Prefer not creating another layer of hierachy (different integrations shouldn't share code).